[FR] Account support team (#5480)
ppy/osu-wiki: wiki/People/The_Team/Account_support_team/fr.md

# Équipe de support aux comptes

> *Pour une équipe qui modère certains sous-forums, voir: [Équipe du support](/wiki/People/The_Team/Support_Team)*

Reviewer: ```suggestion
*Pour l'équipe qui modère certains sous-forums, voir : [Équipe du support](/wiki/People/The_Team/Support_Team)*
```

> **L'équipe de support aux comptes** est un groupe d'utilisateurs administrant les comptes utilisateurs et les appels concernant les silences et les restrictions.

Reviewer: ```suggestion
L'**équipe de support aux comptes** est un groupe d'utilisateurs administrant les comptes utilisateurs et les demandes appels concernant les silences et les restrictions.
```

## Contacter l'équipe

Le **seul** moyen de contacter cette équipe est d'envoyer un mail à [support@ppy.sh](mailto:support@ppy.sh), [accounts@ppy.sh](mailto:accounts@ppy.sh), ou [privacy@ppy.sh](mailto:privacy@ppy.sh). L'identité des membres de cette équipe est gardée secrète afin de leur éviter une attention indésirable.

> Cela prend habituellement quelques jours avant que l'équipe arrive à votre mail. Si vous ne recevez pas de réponse en moins d'une semaine, vous pouvez renvoyer un message sous le premier.

Reviewer: ```suggestion
Cela prend habituellement quelques jours avant que l'équipe prête attention à votre mail. Si vous ne recevez pas de réponse en moins d'une semaine, vous pouvez renvoyer un message sous le premier.
```

## Rôle et tâches

> *Article principal: [Centre d'Aide](/wiki/Help_Centre)*

Reviewer: ```suggestion
*Page principale : [Centre d'aide](/wiki/Help_Centre)*
```

> *Voir également: [Restriction de comptes](/wiki/Help_Centre/Account_Restrictions)*

Reviewer: ```suggestion
*Voir également : [Restriction de comptes](/wiki/Help_Centre/Account_Restrictions)*
```

> Vous devriez contacter cette équipe pour les sujets concernant votre compte qui sont hors de votre contrôle. Veuillez fournir le plus de contexte possible sur votre problème et envoyer un mail à l'adresse associée à votre problème.

Reviewer: ```suggestion
Vous devriez contacter cette équipe pour les sujets concernant votre compte qui sont hors de votre contrôle. Veuillez fournir le plus de détails possible sur votre problème et envoyer un mail à l'adresse associée à votre problème.
```

### [accounts@ppy.sh](mailto:accounts@ppy.sh)

> - [Appel concernant les restrictions placées sur votre compte](/wiki/Help_Centre/Account_Restrictions), incluant:

Reviewer: ```suggestion
- [Appel concernant les restrictions placées sur votre compte](/wiki/Help_Centre/Account_Restrictions), incluant :
```

  - Les silences;
  - Les suppressions de contenu du profil;
  - Les suppressions de beatmaps;

>   - Les suppressions de posts forum et commentaires.

Reviewer: ```suggestion
  - Les suppressions de posts sur le forum et commentaires.
```

> - De la mauvaise conduite que vous [souhaitez admettre](/wiki/Reporting_Bad_Behaviour/Handling_Foul_Play#what-can-i-do-if-i've-broken-the-rules?).

Reviewer: ```suggestion
- De la mauvaise conduite que vous [souhaitez avouer](/wiki/Reporting_Bad_Behaviour/Handling_Foul_Play#what-can-i-do-if-i've-broken-the-rules?).
```

- [Perte d'accès à l'email](/wiki/Help_Centre#sign-in) attaché à votre compte osu!, ou lorsque votre compte a été volé.
- [Rétablissement d'anciens pseudos et corrections de fautes mineures dans votre pseudo](/wiki/Help_Centre#name-changes).
- [Inscriptions de comptes bot](/wiki/Bot_Account).

### [support@ppy.sh](mailto:support@ppy.sh)

> *Note: Pour signaler un cas extrême de mauvaise conduite ou d'abus sexuel de n'importe quel utilisateur (modérateur inclus), voir [Signaler un abus](/wiki/Reporting_Bad_Behaviour/Abuse).*

Reviewer: ```suggestion
*Note : Pour signaler un cas extrême de mauvaise conduite ou d'abus sexuel de n'importe quel utilisateur (modérateur inclus), voir : [Signaler un abus](/wiki/Reporting_Bad_Behaviour/Abuse).*
```

> *Note : Pour signaler un cas extrême de mauvaise conduite ou d'abus sexuel de n'importe quel utilisateur (modérateur inclus), voir : [Signaler un abus](/wiki/Reporting_Bad_Behaviour/Abuse).*

Reviewer: ```suggestion
*Remarque : Pour signaler un cas extrême de mauvaise conduite ou d'abus sexuel de n'importe quel utilisateur (modérateur inclus), voir : [Signaler un abus](/wiki/Reporting_Bad_Behaviour/Abuse).*
```

- Préoccupations concernant la mauvaise conduite d'un modérateur.
- Préoccupations concernant un modérateur brisant [le Code de conduite du contributeur](/wiki/Contributor_Code_of_Conduct).

> - [Problèmes de paiement](/wiki/en/Help_Centre#supporter) pour les commandes du [osu!store](https://osu.ppy.sh/store/listing).

Reviewer: ```suggestion
- [Problèmes de paiement](/wiki/en/Help_Centre#supporter) pour les commandes concernant l'[osu!store](https://osu.ppy.sh/store/listing).
```

- Problèmes techniques n'ayant pas pu être résolus dans le [forum d'aide](https://osu.ppy.sh/community/forums/5), tels que les [problèmes de connectivité](/wiki/Help_Centre#online-features).

### [privacy@ppy.sh](mailto:privacy@ppy.sh)

- [Demandes de données](/wiki/Legal/Privacy#data-controller) et problèmes de confidentialité.

> - [Requêtes de suppression de compte](/wiki/Legal/Privacy#your-rights-and-control) **permanente** pour les utilisateurs résidant dans l'UE ([RGPD](https://fr.wikipedia.org/wiki/Règlement_général_sur_la_protection_des_données "Wikipedia")).

Reviewer: ```suggestion
- [Requêtes de suppression de compte](/wiki/Legal/Privacy#your-rights-and-control) **permanente** pour les utilisateurs résidant dans l'Union Européenne ([RGPD](https://fr.wikipedia.org/wiki/Règlement_général_sur_la_protection_des_données "Wikipédia")).
```
